server/zoom/oauth.go

```diff
@@ -95,15 +101,37 @@ func (c *OAuthClient) GetMeeting(meetingID int) (*Meeting, error) {
 }
 
 func (c *OAuthClient) getUserViaOAuth(user *model.User) (*User, error) {
-	client := c.config.Client(context.Background(), c.token)
 	url := fmt.Sprintf("%s/users/me", c.apiURL)
 	if c.isAccountLevel {
 		url = fmt.Sprintf("%s/users/%s", c.apiURL, user.Email)
+		currentToken, err := c.api.GetZoomSuperUserToken()
+		if err != nil {
+			return nil, errors.Wrap(err, "error getting zoom super user token")
```

Comment:
How does the superuser token get set initially? Is it assumed the superuser has already gone through the OAuth process at

https://github.com/mattermost/mattermost-plugin-zoom/blob/27c408e8f159a0d29cc1ffe8e35e7121026b3472/server/http.go#L123-L129

Reply:
yeah, it is assumed that it is set during the initial handler.

Reply:
yup, this is assumed. sorry for the delay in updates to this, had some personal issues so had to take time off.